app/controllers/v0/profile/transactions_controller.rb

```diff
@@ -15,6 +15,11 @@ def status
         render json: transaction, serializer: AsyncTransaction::BaseSerializer
       end
 
+      def statuses
+        transactions = AsyncTransaction::Vet360::Base.refresh_transaction_statuses(@current_user, service)
+        render json: transactions
```

Comment:
Are we using a named serializer here?

Reply:
It's an array of serialized objects that's being returned. It appears that Rails is matching the items to the correct serializer, but I'll double check. Is there a way to explicitly assign a serializer for the items of an array?

Reply:
Yeah, we'll want to use a serializer here so that we can get that consistent `{ data: attributes: ...}}` response.body structure.